### DISCUSSION TITLE
Binding is not working  properly when using custom delegate and Generic EventHandler for respective properties

### DISCUSSION THREAD

**Discussion by GokulThanu** [dotnet/winforms#7786] [Q&A] [answered]
**Issue Description**

The Binding is not working properly when using  events with  Custom delegate and Generic EventHandler delegate for respective properties.

The Binding is working Properly when using EventHandler delegate

**.NET version**

.net6.0

**Did it work in .NET Framework?**

No.

**Steps to reproduce**

Please find the simple sample in the below github link

https://github.com/GokulThanu/testingsamples/tree/main/Binding%20Issue-detailed/Binding%20Issue

- Run the sample
- Enter any Number in Custom control
- Note that the last textbox is alone updated.

 **Observed**
    The last TextBox is alone updated with the value entered in the CustomControl

 **Expected**
       All TextBox is to be updated with the values entered in the CustomControl

**Comment by KlausLoeffelmann** [comment, marked as answer]
Can you please create a bug issue in the repo for this, and describe the issue with a bit more detail and ideally with a few screenshots?
